answerking_app/tests/test_orderlines.py

```diff
@@ -157,7 +156,7 @@ def test_update_existing_orderline_invalid_returns_bad_request(self):
                 "details": "Failed to add item to order",
             }
         }
-        post_data: OrderIncorrectItemQtyType = {"quantity": "f"}
+        post_data: dict = {"quantity": "f"}
```

Comment:
it should be replaced by the right type

Reply:
and with a comment in front to avoid the typeerror

Reply:
made the changes